content/Rustup-1.28.2.md

```diff
@@ -0,0 +1,84 @@
++++
+path = "2025/04/25/Rustup-1.28.2"
+title = "Announcing rustup 1.28.2"
+authors = ["The Rustup Team"]
+aliases = ["2025/04/25/Rustup-1.28.2.html"]
++++
+
+The rustup team is happy to announce the release of rustup version 1.28.2.
+[Rustup][install] is the recommended tool to install [Rust][rust], a programming language that
+empowers everyone to build reliable and efficient software.
+
+## What's new in rustup 1.28.2
+
+The headlines of this release are:
+
+- The cURL download backend and the native-tls TLS backend are now officially deprecated and
+  a warning will start to show up when they are used. [pr#4277]
+
+  - While rustup predates reqwest and rustls, the rustup team has long wanted to standardize on
+    an HTTP + TLS stack with more components in Rust, which should increase security, potentially
+    improve performance, and simplify maintenance of the project.
+    With the default download backend already switched to reqwest since [2019][pr#1660], the team
+    thinks it is time to focus maintenance on the default stack powered by these two libraries.
+
+  - The rustup team encourages everyone still relying on non-default backends to switch back to
+    reqwest and rustls, and would love to hear from you about your use case via
+    [GitHub Issues][issue tracker] if it does not work well with your particular setup.
```

Comment:
I think this could be clearer still, maybe:

"For people who have RUSTUP_USE_CURL=1 or RUSTUP_USE_RUSTLS=0 set in their environment to work around issues with rustup, please try to unset these after upgrading to 1.28.2 and file an issue [..] if you still encounter problems."

Reply:
Fixed!